website/static/js/project-organizer.js

```diff
@@ -364,10 +375,14 @@ var tbOptions = {
                     }
                     tb.filter(event);
                 },
-                onchange: m.withAttr('value', tb.filterText),
+                onchange: function() {
+                    m.withAttr('value', tb.filterText);
```

Comment:
Yep, I had a poor understanding of when `onchange` was fire. You're right; the `trackClick` belongs here. So I think the only change to make is to replace the `withAttr` with `tb.filterText(event.value)` .